operations/mimir/autoscaling.libsonnet

```diff
@@ -270,36 +276,62 @@
   local cpuHPAQuery(with_ready_trigger) = (
     if with_ready_trigger then
       // To scale out relatively quickly, but scale in slower, we look at the average CPU utilization
```

Comment:
We should update this comment because the way it works is different between rate() and irate(). E.g. in the new query it's **not** the "we look at the average CPU utilization per replica over 5m (rolling window)".